naga/src/compact/expressions.rs

```diff
@@ -233,6 +237,10 @@ impl ExpressionTracer<'_> {
             Ex::ArrayLength(expr) => {
                 self.expressions_used.insert(expr);
             }
+            // CallResult expressions do contain a function handle, but any used
+            // CallResult expression should have an associated `ir::Statement::Call`
```

Comment:
suggestion: Let's be consistent with backticks here:

```suggestion
            // `CallResult` expressions do contain a function handle, but any used
            // `CallResult` expression should have an associated `ir::Statement::Call`
```

Reply:
Pushed as a `fixup!` with 90169042.